common/v2/features/Dashboard/components/AccountList.tsx

```diff
@@ -60,15 +60,13 @@ function buildAccountTable(accounts: ExtendedAccount[], deleteAccount: DeleteAcc
       translateRaw('ACCOUNT_LIST_DELETE')
     ],
     body: accounts.map(account => {
+      const detectedLabel: AddressBook | undefined = getLabelByAccount(account);
+      const label = !detectedLabel ? 'Unknown Account' : detectedLabel.label;
       return [
         // tslint:disable-next-line: jsx-key
         <Icon icon="star" />,
         // tslint:disable-next-line: jsx-key
-        <Address
-          title={`${account.label}-(${account.wallet})`}
-          address={account.address}
-          truncate={truncate}
-        />,
+        <Address title={`${label}`} address={account.address} truncate={truncate} />,
         // tslint:disable-next-line: jsx-key
         <Network color="#a682ff">{account.network}</Network>,
         // tslint:disable-next-line: jsx-key
```

Comment:
The table body is strange, but the disable-next-line stranger (if tslint is complaining react will complain as well during runtime).

How about adding something like this:

```
body: accounts.map((account, idx) => {
 const bodyItemCount = 0;
 return [
     <Icon key={idx + bodyItemCount++} icon="star" />,
      <Address key={idx + bodyItemCount++} title={`${label}`} address={account.address} truncate={truncate} />,
   //....
 ]
}
```